phpunit/functional/TicketTest.php

```diff
@@ -6445,6 +6446,98 @@ public function testAssignFromCategoryOrItem(): void
         }
     }
 
+    public function testAssignFromCategoryOrItemExecuteRule(): void
+    {
+        $tech_id    = getItemByTypeName('User', 'tech', true);
+
+        $entity_id = getItemByTypeName('Entity', '_test_root_entity', true);
+        $this->login();
+
+        $entity = new Entity();
+        $this->assertTrue($entity->update(['id' => $entity_id, 'auto_assign_mode' => Entity::AUTO_ASSIGN_HARDWARE_CATEGORY]));
```

Comment:
Like `createItem()`, you can use `updateItem()`

Reply:
can you apply changes (@RomainLvr)?